content/en/blog/2022/metrics-announcement.md

```diff
@@ -0,0 +1,37 @@
+---
+title: OpenTelemetry Metrics Release Candidates
+date: 2022-05-18
+author: Morgan McLean
+draft: true # TODO: remove this line once your post is ready to be published
+---
+
+OpenTelemetry’s metrics capabilities are now available as release candidates! This means that the specification, APIs, SDKs, and other components that author, capture, process, and otherwise interact with metrics now have the full set of OpenTelemetry metrics functionality and are ready for use. These release candidates will be promoted to general availability throughout the next few weeks.
```

Comment:
> OpenTelemetry’s metrics capabilities are now available as release candidates!

It isn't for all languages, so it might be better to be clear about this from the start. WDYT @mtwo?

At a minimum, consider something like this:

> OpenTelemetry’s ... as release candidates (RCs) -- for most languages, for details, read on.

Or list the languages that are in RC.

Reply:
```suggestion
OpenTelemetry’s metrics capabilities are now available as release candidates, starting with Java, .Net, and Python! This means that the specification, APIs, SDKs, and other components that author, capture, process, and otherwise interact with metrics now have the full set of OpenTelemetry metrics functionality and are ready for use. These release candidates will be promoted to general availability throughout the next few weeks.
```